app/views/alchemy/admin/sites/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= alchemy_form_for site, url: alchemy.admin_sites_path(site, search_filter_params) do |f| %>
-  <%= f.input :host, hint: resource_handler.help_text_for(name: :host).html_safe %>
+  <%= f.input :host, hint: resource_handler.help_text_for(name: :host).try(:html_safe) %>
```

Comment:
I think we should make sure to [always return a String from `help_text_for` instead](https://github.com/AlchemyCMS/alchemy_cms/blob/master/lib/alchemy/resource.rb#L223-L227), actually by not rescuing a exception we force, wdyt?

Reply:
If there is no help text, I'd rather return `nil` than `false` (or any string)... And, in that nil-case, use the safe navigation operator in the view. We still need the exception handler (or a similar solution checking for missing translations) though as no I18n method returns the "string or nil" we need.

Reply:
Agreed 👌